README.md

```diff
@@ -4,6 +4,11 @@ An extremely fast Python type checker and language server, written in Rust.
 
 **This project is still in development and is not ready for production use.**
```

Comment:
Should this be removed?

Reply:
Yeah I think I'd replace this and avoid adding a new heading?

Reply:
I left it in intentionally. I realize it's a bit redundant, but I think it doesn't hurt to be redundant here.

I do think if we use the Warning note to be more prominent, we could probably remove this.

Reply:
Done